packages/styles/global/variables.scss

```diff
@@ -132,79 +132,75 @@
   /* Old Colors
   * TODO: Remove it after all color variables are migrated to new color variables
   * ------------------------- */
-  --black: #111;
-  --white: #fff;
-  --gray-1: #f5f5f5;
-  --gray-2: #d8d8d8;
-  --gray-3: #9b9b9b;
-  --gray-4: #4a4a4a;
-  --gray-5: #333;
-  --blue-1: #daf0ff;
-  --blue-2: #007fd8;
-  --blue-3: #0060a7;
-  --blue-4: #003a64;
-  --green-1: #7ed321;
-  --orange-1: #ff7d00;
+  --black: var(--color-neutral-black);
+  --white: var(--color-neutral-white);
+  --gray-1: var(--color-neutral-light-5);
+  --gray-2: var(--color-neutral-light-3);
+  --gray-3: var(--color-neutral-light-2);
+  --gray-4: var(--color-neutral-light-1);
+  --gray-5: var(--color-neutral-regular);
+  --blue-1: var(--color-brand-secondary-light-2);
+  --blue-2: var(--color-brand-secondary-regular);
+  --blue-3: var(--color-brand-secondary-regular);
+  --blue-4: var(--color-brand-secondary-dark-1);
+  --green-1: var(--color-contextual-success-regular);
+  --orange-1: var(--color-brand-primary-regular);
   --purple-1: #bd10e0;
-  --red-1: #ba1e24;
-  --red-2: #d0021b;
+  --red-1: var(--color-contextual-error-dark-1);
+  --red-2: var(--color-contextual-error-regular);
```

Comment:
**question** @felipefialho i remember these vars were here for compatibility. Once the projects are being updated to atomium, wouldn't be better to remove this old onde and bump te version to X.0.0 as breaking changes?

Reply:
Ps: I turned it into WIP because I edited the wrong file and I'm changing the components variables (.module.scss) instead of this global variables file